lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8392,23 +8392,37 @@ bool SwiftASTContext::CacheUserImports(SwiftASTContext &swift_ast_context,
   return true;
 }
 
+namespace {
+struct CUSignature : public std::pair<Module *, lldb::user_id_t> {
+  CUSignature(CompileUnit &compile_unit)
+      : std::pair<Module *, lldb::user_id_t>(compile_unit.GetModule().get(),
+                                             compile_unit.GetID()) {}
```

Comment:
are `CompileUnit` IDs not unique enough?

Reply:
Nope:

https://lldb.llvm.org/cpp_reference/classlldb__private_1_1CompileUnit.html

>  The UserID of the compile unit is specified by the SymbolFile plug-in and can have any value as long as the value is unique within the Module that owns this compile units.